nih-sparc/sparc-app
# feat(Get Dataset): Provide AWS Download Option For All Datasets

# Description

The purpose of this PR is to provide download messaging/option for all datasets in the Get Dataset dialog.

[Clickup Ticket](https://app.clickup.com/t/cuj2gt)
[Wrike Ticket](https://www.wrike.com/workspace.htm?acc=3203588&wr=14#path=folder&id=441356195&c=list&a=3203588&t=529262647&so=10&bso=10&sd=1&f=&st=space-441356195)

## Type of change


# How Has This Been Tested?

1. Navigate to a dataset that is less than 5GB in size. 
2. Click on the `Get Dataset` button. You should see both the option to download the dataset (since it's less than 5GB) and the AWS option. 
3. Click on `Close`. The dialog should close. 
4. Navigate to a dataset that is greater than 5GB in size. Clicking on the `Get Dataset` button should open the same dialog, only this time the download button isn't available and messaging around that should be changed.
5. View the dialog in different viewports. Sizing should be similar to what's seen on Discover for different viewports.

# Checklist:
